### PR TITLE
Automated SDK generation

### PR DESCRIPTION
**Pull request by lblackstone**
* Preparing the SDK folder for regeneration

* Discovery documents

* Regenerating based on discovery

Fix https://github.com/pulumi/pulumi-google-native/issues/735

### CODE REVIEW

**Comment by lblackstone** [comment]
Hmm, looks like something in this update broke tests. Will need some further work before it's RFR.

File: provider/go.mod [hunk 1]
```diff
@@ -13,8 +13,8 @@ require (
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/mitchellh/mapstructure v1.4.3
 	github.com/pkg/errors v0.9.1
-	github.com/pulumi/pulumi/pkg/v3 v3.46.0
-	github.com/pulumi/pulumi/sdk/v3 v3.46.0
+	github.com/pulumi/pulumi/pkg/v3 v3.45.1-0.20221108173417-4910d9cc9a88
+	github.com/pulumi/pulumi/sdk/v3 v3.46.1-0.20221108173417-4910d9cc9a88
```
Comment (viveklak) [inline provider/go.mod:17]: Hmm what's with the different leading semver between the two but the trailing snapshot version?

**Review by viveklak — APPROVED** [review]
One comment but otherwise LGTM!

File: provider/go.mod (hunk 1 above)
Comment (lblackstone) [reply provider/go.mod:17]: It looks like the pulumi/pulumi `pkg` and `sdk` tags for `v3.46.0` are on different commits, so I think that's why.

**Pull request merged**